{
    # Images published to /navigation_camera/image_raw
    # Image size is 768 x 1024
    "overhead": {
        "nav": {
            # The mask contain params to make a circular mask over the image
            # Pass "mask": null to not create mask
            "mask": null,
            #"mask": {
            #    "width": 768,
            #    "height": 768,
            #    "radius": null,
            #    "center": null # Pass "center": {"x": __, "y": __} to define center
            #},
            # The crop contain params to crop the image
            # x_max - x_min = mask_width and y_max - y_min = mask_height
            # Pass "crop": null to not crop image
            # "crop": {
            #     "x_min": 0,
            #     "x_max": 768,
            #     "y_min": 128,
            #     "y_max": 896
            # },
            "crop": null,
            # Number of degress to rotate video stream clockwise
            # Pass "rotate": null to not rotate
            # "rotate": null,
            "rotate": 'ROTATE_90_CLOCKWISE'
        },
        # "manip": {
        #     "mask": {
        #         "width": 500,
        #         "height": 500,
        #         "radius": null,
        #         "center": null
        #     },
        #     "crop": {
        #         "x_min": 250,
        #         "x_max": 750,
        #         "y_min": 400,
        #         "y_max": 900
        #     },
        #     "rotate": 'ROTATE_90_COUNTERCLOCKWISE'
        # }
    },
    "realsense": {
        "default": {
            "mask": null,
            "crop": null,
            "rotate": 'ROTATE_90_CLOCKWISE'
        }
    },
    # Images published to /gripper_camera/image_raw
    # Image size is 768 x 1024
    "gripper": {
        # "default": {
        #     # The mask contain params to make a circular mask over the image
        #     # Pass "mask": null to not create mask
        #     "mask": {
        #         "width": 768,
        #         "height": 768,
        #         "radius": null,
        #         "center": null
        #     },
        #     # The crop contain params to crop the image
        #     # x_max - x_min = mask_width and y_max - y_min = mask_height
        #     # Pass "crop": null to not crop image
        #     "crop": {
        #         "x_min": 0,
        #         "x_max": 768,
        #         "y_min": 128,
        #         "y_max": 896
        #     },
        #     "rotate": null
        # },
        "d405": {
            # The mask contain params to make a circular mask over the image
            # Pass "mask": null to not create mask
            "mask": {
                "width": 270,
                "height": 270,
                "radius": null,
                "center": null
            },
            # The crop contain params to crop the image
            # x_max - x_min = mask_width and y_max - y_min = mask_height
            # Pass "crop": null to not crop image
            "crop": {
                "y_min": 125,
                "y_max": 395,
                "x_min": 0,
                "x_max": 270
            },
            "rotate": null
        },
    },
}
